dll/win32/comctl32/lang/comctl_Hr.rc

```diff
@@ -0,0 +1,99 @@
+/*
+ * Top level resource file for Common Controls
+ *
+ * Copyright 2022 Andrej Bartulin
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+LANGUAGE LANG_SERBIAN, SUBLANG_DEFAULT
```

Comment:
This needs to be fixed.

Reply:
Uploaded code but still buttons don't work